SalesforceFoundation/EDA
# Multiple affiliations to the same Business Organization prevent Primary field on Contact from updating

# Critical Changes

# Changes
## General Updates
The Primary field on a Contact record is no longer cleared when a new Affiliation with the same Account is identified as Primary for the same Contact. 

For example, as a student at Chapel Hill, Mia Hamm has an Affiliation record that identifies "Chapel Hill Soccer" as her Primary Sports Organization. Following graduation, when she accepts a role as a Faculty member, another Affiliation record for "Chapel Hill Soccer" is created, also with a Primary setting. Mia's Contact record now reflects "Chapel Hill Soccer," with the role of Faculty, as her Primary Sports Organization.

# Issues Closed
Closes #831 

# New Metadata

# Deleted Metadata

# Testing Notes
1. Create a Contact
2. Create an Account with record type “Business organization” or any other record type.
3. Create an Affiliation from Contact created on Step 1 to the “Business Organization” created in step 2 and select Primary. Save. Refresh.
4. Confirm Primary field on Contact is populated
5. Create another Affiliation to the same Business Organization for the same Contact, and select Primary. Save
6. Confirm Bug. Primary field on Contact is blank

With the fix the Primary field retains the value.
